pkg/controller/perconaservermongodbbackup/backup.go

```diff
@@ -91,7 +91,7 @@ func (b *Backup) Start(ctx context.Context, k8sclient client.Client, cluster *ap
 	case api.BackupStorageS3:
 		status.S3 = &stg.S3
 		if stg.S3.Prefix != "" {
-			status.Destination = stg.S3.Prefix + "/"
+			status.Destination = stg.S3.Bucket + "/" + stg.S3.Prefix + "/"
 		}
 	case api.BackupStorageAzure:
 		status.Azure = &stg.Azure
```

Comment:
Can we also do the same for azure?

Reply:
@pooknull For azure we have a container, we don't have a bucket, is that what you mean?

Reply:
@inelpandzic yes

Reply:
Done: [6f9a5f3](https://github.com/percona/percona-server-mongodb-operator/pull/1307/commits/6f9a5f320dd12fbb281463d8f10b7c9dfd577b92)